tools/Generator/Generator.cs

```diff
@@ -396,8 +393,19 @@ bool WriteHandle(XElement handleElement)
 		{
 			string name = handleElement.Element ("name").Value;
 			string csName = GetTypeCsName (name, "struct");
+			typesTranslation [name] = csName;
 
-			writer.WriteLine ("\tpublic class {0}\n\t{{\n\t}}", csName);
+			string parent = null;
+
+			if (handleElement.Attribute ("parent") != null)
+				parent = handleElement.Attribute ("parent").Value
+					.Split (',')
+					.Select (pName => GetTypeCsName (pName))
+					.Aggregate ((a, p) => string.Format ("{0}, {1}", a, p));
+			
+			writer.WriteLine ("\tpublic class {0}{1}\n\t{{\n\t}}",
+				csName,
+				parent == null ? string.Empty : string.Format (" : {0}", parent));
```

Comment:
@radekdoulik This is not good, the Vulkan API suppose multi-hierarchy (VkDisplayModeKHR).